test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -631,4 +634,42 @@ public void testGetDocContent() throws Exception {
     metadata.add("col2",  "John");
     assertEquals(metadata, response.getMetadata());
   }
+
+  @Test
+  public void testClobColumnAsMetadata() throws Exception {
+    String content = "Hello World";
+    executeUpdate("create table data(ID int, CONTENT clob)");
```

Comment:
Why are you uppercasing all the column names? Idiomatic SQL is the other way around, if you want to uppercase something (which I don't).